scte35/state_test.go

```diff
@@ -510,48 +480,41 @@ func TestOutInInOutIn(t *testing.T) {
 		t.Errorf("There should be one open signal (%d)", len(state.Open()))
 	}
 
-	// 0x36 - event_id: 1342177266 - seg_num: 0 - seg_expected: 0
-	// This signal will close the previous 0x36.
-	secondOutSignalBytes, _ := base64.StdEncoding.DecodeString("/DBLAAF0QXOWAP/wBQb+AAAAAAA1AjNDVUVJT///8n//AACky4AJH1NJR05BTDozR1NOanl3cE1sb0FBQUFBQUFBQkFRPT02AAA9gIK2")
-	secondOutSignal, err := NewSCTE35(append([]byte{0x0}, secondOutSignalBytes...))
+	// 0x30 Another Out
+	padOpenSignalBytes, _ := base64.StdEncoding.DecodeString("/DBsAAH/7m1eAAKQBQb+sX6o+wBWAlRDVUVJAAAAJ3//AAApMuANQAwOQU1DTiBMMDAxMjM0NTYJCFBPOjEyMzQ1DiRiY2IxZGQ4ZS1kMzIzLTQ1ODktOWQ3OC1hM2QxMzYyYWJiYjYwAQGtc8xr")
+	padOpenSignal, err := NewSCTE35(append([]byte{0x0}, padOpenSignalBytes...))
 	if err != nil {
 		t.Errorf("Error creating SCTE-35 signal: %s", err.Error())
 	}
 
-	closed, err = state.ProcessDescriptor(secondOutSignal.Descriptors()[0])
+	closed, err = state.ProcessDescriptor(padOpenSignal.Descriptors()[0])
 	if err != nil {
 		t.Errorf("ProcessDescriptor returned an error: %s", err.Error())
 	}
-	if len(closed) != 1 {
-		t.Errorf("One event should have been closed (%d were)", len(closed))
+	if len(closed) != 0 {
+		t.Errorf("No events should have been closed (%d were)", len(closed))
 	}
-	if len(state.Open()) != 1 {
+	if len(state.Open()) != 2 {
 		t.Errorf("There should be one open signals (%d)", len(state.Open()))
```

Comment:
Should this error message be updated to reflect the new conditional?

Reply:
oops geez, bad miss. sorry! will fix!